core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -327,6 +327,24 @@ public T keepAliveWithoutCalls(boolean enable) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Sets the retry buffer size in bytes. All RPCs are not retriable if the buffer limit is
```

Comment:
This doesn't quite seem true. _All_ RPCs are not retriable when it is exceeded; only new ones while it is exceed. How about: "Individual RPCs will disable retries instead of causing the cumulative limit to be exceed." I'd be fine if it was worded a different way.

Reply:
Revised.